joomla/joomla-cms
# Allow a custom page title to be defined for an article without a menu item

#### Summary of Changes

When an article is displayed using a menu item that isn't a single article view, the browser page title is hardcoded to use the article's title if one is defined (and one is required unless you've hacked your database).  This PR makes it possible to customize the browser page title for a case when an article is not displayed using a com_content article menu item (i.e. displaying an article that's a child of a blog with a menu item).  Note that this does NOT overrule the existing `page_title` parameter's logic or alter its use in any way.
#### Testing Instructions

From a default install, create a new article and assign it to the "Uncategorised" category.  Change the home menu item to be a category menu item pointing to this category.  Set the "Browser Page Title" on the "Page Display" tab to whatever you'd like.

Pre-patch, navigate to the frontend and see that the page title for the article is the article's title.

With the patch applied, on the options tab of the backend Article Manager's edit view, you'll see a new "Browser Page Title" field.  Set this to whatever you'd like.

On the frontend, refresh the page and you should now see the browser's page title is the value you set in this new field.
